microsoft-edge/webview2/concepts/browser-features.md

```diff
@@ -129,8 +136,8 @@ The following shortcuts are always turned off in WebView2.  An asterisk (`*`) in
 | Select Tab (1 - 8) | `Ctrl`+`(1-8)` |
 | Show Favorites Bar `*` | `Ctrl`+`Shift`+`B` |
 | Help | `F1` |
-| Focus Next Pane `*` | `F6` |
-| Focus Previous Pane `*` | `Shift`+`F6` |
+| Focus Next Pane `*` | `F6`.  Supported in windowed hosting mode, but not in visual hosting mode. |
```

Comment:
This looks good to me but I'm not sure if these hosting modes are known concepts to end devs. Maybe add some more information like: `Supported in windowed hosting mode, but not in visual hosting mode (UWP WinUI 2 and Win32 C++ app with Visual Composition)`.  These are what we put in our docs here https://docs.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2_sample_uwp so maybe that'll make it clearer for devs to look for information.